src/Collections/Callbacks.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Swagger package.
+ *
+ * (c) EXSyst
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace EXSyst\OAS\Collections;
+
+use EXSyst\OAS\AbstractObject;
+use EXSyst\OAS\Callback;
+use function EXSyst\OAS\referenceOr;
+
+/**
+ * Helper class - Does not exist in actual Spec.
+ */
+final class Callbacks extends AbstractObject
+{
+    private $callbacks = [];
```

Comment:
May deserve a type hint :) `@var Callback[]|Reference[]`

Reply:
@GuilhemN yeah sure, just being lazy. Infact it could be added to code sniffer